### PR TITLE
Convert Live Order Display to table format. 

### PR DESCRIPTION
**Pull request by zachpren0**
Convert live order display to table format. 

Added CSS styling to fit table to window and size correctly.

Added Scrum Minutes for the week

### CODE REVIEW

**Review by SpeiserK — APPROVED** [review]
Good work styling live order.
Had to remove the smiley face 👎

**Review by camwlsn — APPROVED** [review]
Nice job fixing up the styling. Looks great!

File: MERN Scripts/mern-emsystem/src/pages/CashConfirmation.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ const navigate = useNavigate();
             </div>
             <div className="rightDiv">
                 <div className="progBar">
-                    <h4 className="progressTitle">Complete :D</h4><br></br>
+                    <h4 className="progressTitle">Complete</h4><br></br>
```
Comment (amurph24) [inline MERN Scripts/mern-emsystem/src/pages/CashConfirmation.js:34]: bruh

**Review by amurph24 — COMMENTED** [review]
good stuff

**Pull request merged**